Documents/Management/WeeklyReports/mainWeeklyReport.md

```diff
@@ -0,0 +1,76 @@
+
+# mainWeeklyReport
+
+<details>
+<summary>Table of contents</summary>
+
+- [Week 1](#weeklyreportfirstweek)
+
+# WeeklyReportFirstWeek
+
+[Week 1](https://github.com/algosup/2023-2024-project-4-sportshield-team-2/Documents/Management/weeklyReportFirstWeek.md)
```

Comment:
line 11: this link is dead.

![Goofy meme](https://image.spreadshirtmedia.net/image-server/v1/compositions/T6A1PA5835PT17X29Y62D162120215W24929H23114/views/1,width=550,height=550,appearanceId=1,backgroundColor=FFFFFF,noPt=true/dead-link-shirt-t-shirt-homme.jpg)